### PR TITLE
comp error fun

### PR DESCRIPTION
**Pull request by RiteshKB**

### CODE REVIEW

File: symengine/functions.cpp [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (certik) [inline symengine/functions.cpp]: Is there a specific reason to have an empty line at the beginning of the file? If not, then I would remove it.

File: symengine/functions.h [hunk 2]
```diff
@@ -924,6 +924,30 @@ class Erf : public OneArgFunction
 //! Canonicalize Erf:
 RCP<const Basic> erf(const RCP<const Basic> &arg);
 
+class Erfc : public OneArgFunction
+{
+    /*   The complementary error function. This function is defined as:
+     *
+     *   .. math::
+     *      \mathrm{erfc}(x) = 1 - \frac{2}{\sqrt{\pi}} \int_0^x e^{-t^2}
+     *	 \mathrm{d}t.
```
Comment (certik) [inline symengine/functions.h]: I would put a few more spaces before `\mathrm{d}t`.

File: symengine/derivative.cpp [hunk 3]
```diff
@@ -427,6 +427,13 @@ class DiffImplementation
                    arg->diff(x));
     }
 
+    static RCP<const Basic> diff(const Erfc &self, const RCP<const Symbol> &x)
+    {
+        RCP<const Basic> arg = self.get_args()[0];
+        return neg(mul(div(mul(integer(2), exp(neg(mul(arg, arg)))), sqrt(pi)),
+                       arg->diff(x)));
```
Comment (certik) [inline symengine/derivative.cpp]: In the test suite, we should test this. To ensure it is correct.

File: symengine/functions.cpp (hunk 1 above)
Comment (RiteshKB) [inline symengine/functions.cpp]: I will remove the space.

**Comment by isuruf** [comment]
Looks good, can you add `erfc` to `eval_mpfr` as well ?

**Comment by isuruf** [comment]
Can you apply the diff here, https://travis-ci.org/symengine/symengine/jobs/169689343#L665-L683 ?
Otherwise +1 to merge

**Comment by RiteshKB** [comment]
can you tell me, how to apply diff?

**Comment by isuruf** [comment]
Save the diff in a file and do `git apply file.txt`.

Or you can use `clang-format-3.7` to format your changes. `apt-get install clang-format-3.7` if you are on debian. And then run, `./bin/test_format_local.sh`

**Comment by RiteshKB** [comment]
@isuruf  can you please review the work. 
all tests have passed.

**Comment by isuruf** [comment]
Thanks for the PR

**Comment by certik** [comment]
Thanks @RiteshKB. Very good PR.

One thing that you can improve next time is to write better git commit messages. There are many guides online how to do that, e.g.:

http://chris.beams.io/posts/git-commit/

Also it helps to squash the commits, to create a nice few logical commits, with good commit logs. That helps with somebody digs in the git history, trying to figure out what was done. Or when using git bisect.

The quality of the changes themselves is very good, thanks again for your work.

**Pull request merged**